template.yaml

```diff
@@ -644,6 +644,23 @@ Resources:
         Variables:
           EVENT_BUS_NAME: !GetAtt InternalBus.Name
 
+  DeleteImportedBragePublicationHandler:
+    Type: AWS::Serverless::Function
+    Properties:
+      Description: 'Should set publication status to deleted'
```

Comment:
This is placed in the wrong section of the template file. Should be placed with the other "Button" lambdas and not the Event lambdas.

Reply:
fixed